### ISSUE TITLE
API endpoint

### ISSUE THREAD

**Issue by andrerfneves** [fiatjaf/satdress#16]
It would be great if the exact same form that is sent through the UI in order to claim a Lightning Address, could also be sent over a very simple JSON api. Something like "/api/v1/claim" or whatever you want to call it. This would allow services and wallets to program this on behalf of the customers/users.

**Comment by fiatjaf** [comment]
We can transition into a full JSON API + static HTML + React if you prefer. Once things get past "the single page prototype" we must do it necessarily anyway.

**Comment by andrerfneves** [comment]
Yeah or maybe we just do NextJS which can do SSR React + JSON APIs out of the box. We dont need TONS of new pages just 2-3 more. And then the API would be a nice to have for automation. 

That said not sure jow the go backend will handle the nextjs stuff so maybe just a simple json api and a create-react-app feontend is the answer.
